src/liberty/devCommands.ts

```diff
@@ -441,11 +441,11 @@ export async function startContainerDevMode(libProject?: LibertyProject | undefi
         if (terminal !== undefined) {
             terminal.show();
             libProject.setTerminal(terminal);
-            if (libProject.getContextValue() === LIBERTY_MAVEN_PROJECT_CONTAINER) {
+            if (libProject.getContextValue() === LIBERTY_MAVEN_PROJECT_CONTAINER || libProject.getContextValue() === LIBERTY_MAVEN_PROJECT ) {
```

Comment:
only LIBERTY_MAVEN_PROJECT_CONTAINER is valid here

Reply:
Reverted the changes